lib/fencing/st_client.c

```diff
@@ -1070,7 +1070,7 @@ stonith_api_status(stonith_t * stonith, int call_options, const char *id, const
 
 static int
 stonith_api_fence(stonith_t * stonith, int call_options, const char *node, const char *action,
-                  int timeout, int tolerance)
+                  int timeout, int tolerance, int delay)
```

Comment:
I see a few approaches to keep backward compatibility:

* Make a second version of every function in stonith_api_operations_t, with the original becoming a wrapper for it.
* Same as above, but instead of adding new entries to stonith_api_operations_t, use some api version flag to know which set of functions to assign to a newly created instance. @wenningerk actually suggested this technique in another situation, and I didn't there would be enough external users to make it worthwhile, but this changes my mind. :)
* Keep the delay in stonith_t. Hacky but simple.

Reply:
Hrm, probably the clearest way for consumers is just introduce a new function...